carbonj.service/src/main/resources/application.yml

```diff
@@ -11,7 +11,8 @@ spring:
     active: dev
 metrics:
   store:
-    enabled: false
+    enabled: true
```

Comment:
Let's leave the default to be disabled here.

Reply:
Done